### DISCUSSION TITLE
Form validation rule for equal field value ?

### DISCUSSION THREAD

**Discussion by chmike** [fomantic/Fomantic-UI#1338] [Behaviors (Api, Form Validation, Visibility)] [answered]
There are various examples of form field validation rules but none for testing that the value of two fields are equal.

The use case is to ensure that a re-entered password or email in a second field is the same as the first one. 

Is this even possible with Fomantic-ui ?

**Comment by lubber-de** [comment, marked as answer]
Yes, this is available.
See https://fomantic-ui.com/behaviors/form.html#matching-fields
![image](https://user-images.githubusercontent.com/18379884/74925298-3aa0a480-53d4-11ea-880a-82e3bb98865d.png)

**Comment by chmike** [comment]
Sorry. I missed that list of rules. It works perfectly.  Thank you.
